web/cobrands/zurich/layout.scss

```diff
@@ -30,9 +30,16 @@ body.mappage {
     #site-header {
         // The header on a map page needs a shadow too
         @include box-shadow(0 0 6px 1px #000);
-        z-index: 2; // One more than #zurich-main-nav so it's on top
+        z-index: auto; // So that #zurich-main-nav rises to the top
         .container {
-            margin: 0 1em; // White padding left/right
+            margin: 0; // White padding left/right
+        }
+        .header__logobar {
+            padding-left: 0;
+            margin-left: 0;
```

Comment:
This line is a duplicate of higher up.

Reply:
As well as removing the duplicate line, you've removed the padding line which was bringing the logo on a map page over to the side to match the content, was that deliberate?

Reply:
Just added padding back in - to align logo left hand content.